src/css/site.styl

```diff
@@ -84,9 +84,11 @@ input, select
   width: 100vw
   top: 0
   left: 0
-  z-index: 500
+  z-index: 400
   transition: color 300ms
-
+  nav
+    position: relative
+    z-index: 400
```

Comment:
the point is so the white line in the header thats blocking the blog border line doesnt run over the footer. so the footer is 300. the nav is 400 and the line is 200 . for some reason i also had to set the nav at 400 to make it work.